### PR TITLE
Add Modern Standard Arabic (ar-001)

### PR DESCRIPTION
**Pull request by onderay**
# Explain your changes

_Suppose there is a related issue with enough detail for a reviewer to understand your changes fully. In that case, you can omit an explanation and instead include either “Fixes #XX” or “Updates #XX” where “XX” is the issue number._

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

### CODE REVIEW

File: ar-001/auth.json [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+            "code_input_error_incorrect_code_length": "يجب أن يتكون رمز التأكيد من 6 أرقام",
+            "code_input_error_resend_requested_too_soon": "لا يمكن إرسال رمز آخر بعد.  الرجاء إعادة المحاولة في وقت لاحق",
+            "confirmation_code_input_error_unknown_error": "خطأ في المصادقة",
+            "continue_button": "تحقق",
+            "didnt_receive_a_code_description": "لم تستلم رمز ؟",
+            "didnt_receive_a_code_link": "إعادة إرسال الرمز ",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "verification_code_sms": {
+            "description": "\"${generated code} هو الرمز الذي تستخدمه لمرة واحدة لتسجيل الدخول إلى ${business_name} @ ${receiver_domain} # ${generated code]\""
```
Comment (DaveOrDead) [inline ar-001/auth.json:227]: @onderay some rogue characters at the end here `]\` also whitespace between `@` and `#` - the format is quite particular for SMS to be picked up correctly. Also there are two sets of `"` which I suspect will break.

```
"${generated code} هو الرمز الذي تستخدمه لمرة واحدة لتسجيل الدخول إلى\n\n${business_name} @${receiver_domain} # ${generated code}"
```

File: ar-001/auth.json [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+            "phone_input_label": "رقم الهاتف",
+            "phone_input_error_required": "أدخل رقم الهاتف",
+            "phone_input_error_invalid_format": "أدخل الأرقام فقط بدون مسافات، باستثناء الصفر البادئ",
+            "phone_input_error_invalid_region": "الرقم غير صالح لهذه المنطقة",
+            "phone_input_help_text": "اترك الصفر البادئ",
+            "continue_button": "المتابعة",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "sms_verify_code_page": {
+            "page_title": "مصادقة متعددة العوامل |  ${business_name}",
```
Comment (DaveOrDead) [inline ar-001/auth.json:212]: @onderay there looks like additional whitespace here on the right hand side of ` | `

File: ar-001/auth.json [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+            "continue_button": "تحقق",
+            "didnt_receive_a_code_description": "لم تستلم رمز ؟",
+            "didnt_receive_a_code_link": "إعادة إرسال الرمز ",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "verification_code_sms": {
+            "description": "\"${generated code} هو الرمز الذي تستخدمه لمرة واحدة لتسجيل الدخول إلى ${business_name} @ ${receiver_domain} # ${generated code]\""
+        },
+        "choose_organization_page": {
+            "page_title": "اختر مؤسسة |  ${business_name}",
```
Comment (DaveOrDead) [inline ar-001/auth.json:230]: @onderay there looks like additional whitespace here on the right hand side of ` | `

File: ar-001/auth.json [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+            "email_input_label": "بريد إلكتروني",
+            "email_input_error_invalid": "أدخل بريد إلكتروني صحيح",
+            "email_input_error_required": "مطلوب بريد إلكتروني",
+            "email_input_error_duplicate_email_found": "هذا البريد الإلكتروني مشترك بالفعل",
+            "continue_button": "حساب جديد",
+            "success_heading": "تم بنجاح! مرحبا بك في الحلقة.",
+            "credit_powered_by": "مدعوم من طرف"
+        },
+        "recovery_code_verify_page": {
+            "page_title": "إستخدم رمز الاسترداد |  ${business_name}",
```
Comment (DaveOrDead) [inline ar-001/auth.json:271]: @onderay there looks like additional whitespace here on the right hand side of ` | `

File: ar-001/auth.json [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+            "code_input_label": "رمز الإستخدام لمرة واحدة ",
+            "code_input_error_required": "الرجاء إدخال رمز التحقق",
+            "code_input_error_incorrect_code_length": "يجب أن يتكون رمز التأكيد من 6 أرقام",
+            "code_input_error_incorrect_code": "الرجاء إدخال رمز تأكيد صحيح",
+            "code_input_error_expired_code": "لقد انتهت صلاحية رمز التأكيد الخاص بك، يرجى طلب رمز جديد",
+            "continue_button": "المتابعة",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "mfa_sms_page": {
+            "page_title": "أدخل رقم الهاتف |  ${business_name}",
```
Comment (DaveOrDead) [inline ar-001/auth.json:198]: @onderay few of these, I won't list them all

**Review by DaveOrDead — CHANGES_REQUESTED** [review]

File: ar-001/auth.json (hunk 1 above)
Comment (onderay) [reply ar-001/auth.json]: @DaveOrDead updated

File: ar-001/auth.json (hunk 2 above)
Comment (onderay) [reply ar-001/auth.json]: @DaveOrDead updated

File: ar-001/auth.json (hunk 3 above)
Comment (onderay) [reply ar-001/auth.json]: @DaveOrDead updated

File: ar-001/auth.json [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+            "password_input_error_too_short": "يجب أن تتكون كلمات المرور من 8 أحرف على الأقل",
+            "continue_button": "المتابعة ",
+            "cant_sign_in_description": "لا تستطيع تسجيل الدخول ؟",
+            "cant_sign_in_link": "إعادة تعيين كلمة المرور ",
+            "password_input_error_unknown_error": "خطأ في المصادقة"
+        },
+        "account_locked_page": {
+            "page_title": "الحساب مقفل | ${business_name}",
+            "heading": "الحساب مقفل",
+            "description": "لقد حاولت تسجيل الدخول عدة مرات وتم قفل حسابك لمدة ${lockout Dur Minions} دقيقة",
```
Comment (evgenyk) [inline ar-001/auth.json:137]: @onderay lockout Dur Minions ;), should be `${lockout_duration_minutes}`

File: ar-001/auth.json [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+            "email_input_label": "البريد الإلكتروني",
+            "email_input_error_invalid": "الرجاء إدخال بريد إلكتروني صحيح ",
+            "email_input_error_required": "مطلوب بريد إلكتروني ",
+            "email_input_error_invitation_mismatch": "يجب أن يتطابق البريد الإلكتروني مع البريد الذي تم إرسال رسالتك إليه",
+            "email_input_error_no_account_found": "لم يتم العثور على حساب بهذا البريد الإلكتروني",
+            "auth_form_error_no_auth_methods_enabled": "عذرًا، لا نرى وسيلة للتحقق من هويتك في الوقت الحالي",
+            "continue_with_email_button": "المتابعة",
+            "auth_block_separator": "أو ",
+            "continue_with_social_provider_button": "المتابعة مع ${social_connection_name]",
+            "continue_with_enterprise_provider_button": "المتابعة مع ${enterprise Connection_name]",
```
Comment (evgenyk) [inline ar-001/auth.json:57]: @onderay `${enterprise_provider_name}`

File: ar-001/auth.json [hunk 8]
```diff
@@ -0,0 +1,283 @@
...
+            "code_input_error_incorrect_code_length": "يجب أن يتكون رمز التأكيد من 6 أرقام",
+            "code_input_error_resend_requested_too_soon": "لا يمكن إرسال رمز آخر بعد.  الرجاء إعادة المحاولة في وقت لاحق",
+            "confirmation_code_input_error_unknown_error": "خطأ في المصادقة",
+            "continue_button": "تحقق",
+            "didnt_receive_a_code_description": "لم تستلم رمز ؟",
+            "didnt_receive_a_code_link": "إعادة إرسال الرمز ",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "verification_code_sms": {
+            "description": "\${generated code} هو الرمز الذي تستخدمه لمرة واحدة لتسجيل الدخول إلى ${business_name} @${receiver_domain} #${generated code}""
```
Comment (evgenyk) [inline ar-001/auth.json:227]: @onderay 
`\$` -> `$`
`""` -> `"`
`generated code` -> `generated_code`

File: ar-001/auth.json [hunk 9]
```diff
@@ -0,0 +1,283 @@
...
+            "page_title": "إستخدم رمز الاسترداد | ${business_name}",
+            "heading": "رمز الإسترداد",
+            "description": "أدخل أحد رموز الاسترداد التي تم توفيرها لك أثناء التسجيل الأولي.",
+            "recovery_code_input_label": "رمز الإسترداد",
+            "continue_button": "المتابعة",
+            "choose_another_method_link": "إختر طريقة أخرى",
+            "recovery_code_input_error_unknown_error": "هناك مشكلة في المصادقة باستخدام رمز الاسترداد هذا",
+            "recovery_code_input_error_required": "أدخل بريدك الإلكتروني الخاص لإسترداد الحساب ",
+            "recovery_code_input_error_too_short": "أدخل رمز إسترداد يحتوي على الأقل على 9 أحرف",
+            "recovery_code_input_error_invalid": "أدخل رمز إسترداد صحيح "
```
Comment (evgenyk) [inline ar-001/auth.json:280]: @onderay looks to have an extra space

File: ar-001/auth.json [hunk 10]
```diff
@@ -0,0 +1,283 @@
...
+            "page_title": "تعيين كلمة المرور | ${business_name}",
+            "heading": "إعداد كلمة المرور",
+            "description": "يرجى تعيين كلمة المرور الخاصة بك",
+            "password_input_label": "كلمة المرور",
+            "password_input_error_extra_space": "لا يمكن أن تبدأ كلمة المرور أو تنتهي بمسافة",
+            "password_input_error_too_common": "هذه كلمة المرور شائعة جدًا. اجعلها أكثر صعوبة في التخمين.",
+            "password_input_error_too_short": "يجب أن تتكون كلمات المرور من 8 أحرف على الأقل",
+            "confirm_password_input_label": "تأكيد كلمة المرور",
+            "confirm_password_input_error_mismatch": "تأكد من أن كلمات المرور متطابقة",
+            "continue_button": " المتابعة",
```
Comment (evgenyk) [inline ar-001/auth.json:118]: @onderay does it needs a space?

File: ar-001/auth.json (hunk 6 above)
Comment (onderay) [reply ar-001/auth.json]: @evgenyk updated

File: ar-001/auth.json (hunk 7 above)
Comment (onderay) [reply ar-001/auth.json]: @evgenyk updated

File: ar-001/auth.json (hunk 8 above)
Comment (onderay) [reply ar-001/auth.json]: @evgenyk updated

File: ar-001/auth.json (hunk 9 above)
Comment (onderay) [reply ar-001/auth.json]: @evgenyk updated

File: ar-001/auth.json (hunk 10 above)
Comment (onderay) [reply ar-001/auth.json]: @evgenyk updated

**Review by evgenyk — APPROVED** [review]
LGTM

**Review by DaveOrDead — APPROVED** [review]

**Pull request merged**